### PR TITLE
MIXED: New feature structure model statistics page, build and db changes. New feature generalised gray scale for tables. Fixes for browsers. Alignment function to remove non-gn residues from alignment

### PR DESCRIPTION
**Pull request by pszgaspar**

### CODE REVIEW

**Review by esguerra — APPROVED** [review]
Looks good.
Couldn't find any issues.
Would like to be able to run the build script locally to test, but something breaks in the migration.

File: build_gpcr/management/commands/build_structure_model_rmsd.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from build.management.commands.build_structure_model_rmsd import Command as StructureModelRMSD
+
+
+class Command(StructureModelRMSD):
+    pass
```
[the remark this answers is not in the document]
Comment (pszgaspar) [reply build_gpcr/management/commands/build_structure_model_rmsd.py:5]: yes, good point. I also don't know why we have that

File: static/home/js/grayscale.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+function componentToHex(c) {
+    var hex = c.toString(16);
+    return hex.length == 1 ? "0" + hex : hex;
+}
+
+function rgbToHex(r, g, b) {
+    return "#" + componentToHex(r) + componentToHex(g) + componentToHex(b);
+}
```
[the remark this answers is not in the document]
Comment (pszgaspar) [reply static/home/js/grayscale.js:108]: Yes, that was the idea to just have the table as input, you would just add class names to <td> objects for which you want grayscale and the script would do the rest. However, Codacy is complaining about complexity and generic object injection sinks, so don't know how it's gonna end up.

[the remark this answers is not in the document]
Comment (pszgaspar) [reply static/home/js/grayscale.js:123]: Added reverse coloring as an option and now values are normalized so the coloring is more diverse

**Pull request merged**